notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb

```diff
@@ -446,6 +446,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Only include datapoints whose ids are 0-100 in the dataset\n",
+    "# Add allow_list 0-100 so that only datapoints whose ids are in 0-100 are allowed to show up in the output\n",
+    "# Split datapoins into two groups 'a' and 'b'. The datapoint whose ids are even are in group 'a', otherwise are in group 'b'\n",
```

Comment:
Let's add "We will demo how to configure the query to return up to k data points for each group."

Reply:
Added.